### DISCUSSION TITLE
Disable attempt to download fonts when running flextable() and use local fonts instead?

### DISCUSSION THREAD

**Discussion by aske-skov** [davidgohel/flextable#561] [Q&A] [answered]
Hello

Thank you for a great package (if the author is reading this).

I'm working in a closed (no access to the internet) corporate R environment and I believe this is the reason that I keep getting the below error message when trying to run the flextable() function.

> Error in curl::curl_fetch_memory(x$url$url, handle = x$url$handle) : 
>   Timeout was reached: [gwfh.mranftl.com] Failed to connect to gwfh.mranftl.com port 443 after 7502 ms: Timed out

According to IT the error occurs because flextable by default tries to send a call to a remote server in order to import fonts.

Since I'm working in a closed environment and have lots of fonts locally available I would like to disable this default behaviour of the flextable() function and use local fonts instead.

Does anyone know a way to alter the behaviour of flextable() and fix this issue?

Best,
Aske

**Comment by davidgohel** [comment, marked as answer]
Could you check you have the latest versions of flextable (and gdtools)? 

This should have been fixed with #525

**Reply by aske-skov** [reply]
Thank you for getting back to me so swiftly!
As it turns out we're currently running version 0.9.0 of Flextable not 0.9.2.

I will have IT update the package. Thank you very much.
